spec/features/smoke_test_url_spec.rb

```diff
@@ -2,11 +2,18 @@
 
 module Kapnismology
   RSpec.feature 'Access to the smoke test API', type: :feature do
-    let(:data) { [:berserk] }
+    let(:data) { { title: 'berserk' } }
     let(:passed) { true }
     let(:message) { '黄金時代' }
     let(:result) { Result.new(passed, data, message) }
     let(:name) { 'guts' }
+    let(:trace_id) { '74bf7dd481bcc880' }
+    let(:count) { 1 }
+    let(:links) { {
+        self: "http://www.example.com/smoke_test#{parameters.gsub(',', '%2C')}",
```

Comment:
I thought so, but `CGI.escape` works too hard:
```ruby
request.original_url
# => http://www.example.com/smoke_test?skip=junk%2Cne

CGI.escape('?skip=junk,ne')
# => "%3Fskip%3Djunk%2Cne"
```

I notice that we should `CGI.unescape(request.original_url)` in `SmokeTestsController`
will modify it